test/test-775/main.js

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const path = require('path');
+const assert = require('assert');
+const utils = require('../utils.js');
+
+assert(!module.parent);
+assert(__dirname === process.cwd());
+
+// note : worker_threads in only valid for nodejs >=12.0
+const MAJOR_VERSION = parseInt(process.version.match(/v([0-9]+)/)[1], 10);
```

Comment:
We could think about adding semver as a project dep and replace this and others with semver.major semver.minor functions